divspl/main.py

```diff
@@ -22,7 +22,8 @@ def _bytes(x):
     if len(argv) > 1:
         with open(argv[1], 'r') as f:
             result = parser.parse(lexer.lex(f.read()))
-            os.write(1, _bytes(result.eval()))
+            for number in result.eval():
+                os.write(1, _bytes(number))
```

Comment:
That's what I thought as well, but when running it the newlines still appeared for me